app/common/AreaFilterSelect.jsx

```diff
@@ -42,18 +42,40 @@ class AreaFilterSelect extends React.Component {
       return null;
     };
 
-    return (
-      <div className="area-filter-select">
-        <Select
-          name="area-filter-select"
-          placeholder="Filter by Administrative Boundary"
-          value={this.state.selectedLayer}
-          options={this.props.options}
-          onChange={this.handleSelectChange}
-        />
-        { polygonSelectorComponent() }
-      </div>
-    );
+    const {
+      initialComponentMount,
+      ntaSelection,
+    } = this.props;
+
+    if (initialComponentMount) {
+      this.state.selectedLayer = { value: 'nta2020', label: 'Neighborhood Tabulation Area' };
+
+      return (
+        <div className="area-filter-select">
+          <Select
+            name="area-filter-select"
+            placeholder="Filter by Administrative Boundary"
+            value={this.state.selectedLayer}
+            options={this.props.options}
+            onChange={this.handleSelectChange}
+          />
+          { polygonSelectorComponent() }
+        </div>
+      );
+    } else {
+      return (
+        <div className="area-filter-select">
+          <Select
+            name="area-filter-select"
+            placeholder="Filter by Administrative Boundary"
+            value={this.state.selectedLayer}
+            options={this.props.options}
+            onChange={this.handleSelectChange}
+          />
+          { polygonSelectorComponent() }
+        </div>
+      );
+    }
   }
 }
 
```

Comment:
This looks a bit awkward but is effective in populating the select boxes after transitioning from the` SplashDuo` component. I tried a ternary do avoid the `if else` block but it failed spectacularly. The options box (on the map page) became stuck on the `Neighborhood Tabulation Area` option.

```
 const {
      initialComponentMount, // set as true in LayerSelector's ComponentDidMount
      ntaSelection,
    } 
const initialSelectedLayer = { value: 'nta2020', label: 'Neighborhood Tabulation Area' };
/* ... */
<Select
  value={initialComponentMount  ? initialSelectedLayer : this.state.selectedLayer}
```